README.md

```diff
@@ -232,6 +232,7 @@ Keys supported by image output:
 * `registry.insecure=true`: push to insecure HTTP registry
 * `oci-mediatypes=true`: use OCI mediatypes in configuration JSON instead of Docker's
 * `unpack=true`: unpack image after creation (for use with containerd)
+* `ensure-blobs=true`: ensure blobs of the result image are locally available
```

Comment:
"locally" needs to be explained

Reply:
Thank you for the comment. Fixed the explanation to "exist in the content store" instead of "locally available".